README.md

```diff
@@ -37,7 +43,24 @@ The easiest way to try the web app is to run it through docker:
 - npm
 - HDF5 and NetCDF4 headers, if available on your platform
 
-On Debian, e.g., install these packages:
+
+### MacOS
+You can install these dependencies using brew. Don’t know what [brew](http://brew.sh/) is?
+
+Run:
+`
+brew install supervisor nginx postgresql npm hdf5
+`
+
+An alternative to installing these dependencies is to use pip. You can install each package by using the following:
```

Comment:
I don't think I'd bother describing that, people can run it however they like...just mention that `supervisor` is Python 2 only but can be installed/run in a separate Python 2 environment.